lib/rubocop/cop/lint/unreachable_loop.rb

```diff
@@ -70,7 +76,16 @@ module Lint
       #     raise NotFoundError
       #   end
       #
+      #   # bad
+      #   2.times { raise ArgumentError }
+      #
+      # @example IgnoredPatterns: [/exactly\(\d+\)\.times/]
```

Comment:
I think this cop can ignore `exactly`, `at_least`, and `at_most` by default. I think it's much less likely that these will be false negatives in non-RSpec context than false positives in RSpec context.

```suggestion
      # @example IgnoredPatterns: [/(exactly|at_least|at_most)\(\d+\)\.times/] (default)
```

Reply:
@koic sounds good! There's a whole bunch more methods that it picks up but this is a good start.

(Currently: `all?`, `any?`, `chain`, `chunk`, `chunk_while`, `collect`, `collect_concat`, `count`, `cycle`, `detect`, `downto`, `drop`, `drop_while`, `each`, `each_cons`, `each_entry`, `each_slice`, `each_with_index`, `each_with_object`, `entries`, `filter`, `find`, `find_all`, `find_index`, `first`, `flat_map`, `grep`, `grep_v`, `group_by`, `include?`, `inject`, `lazy`, `loop`, `map`, `map!`, `max`, `max_by`, `member?`, `min`, `min_by`, `minmax`, `minmax_by`, `none?`, `one?`, `partition`, `reduce`, `reject`, `reject!`, `reverse_each`, `select`, `select!`, `slice_after`, `slice_before`, `slice_when`, `sort`, `sort_by`, `sum`, `take`, `take_while`, `times`, `to_a`, `to_h`, `to_set`, `uniq`, `upto`, `zip`)

Reply:
Updated!